src/LibraryUtilities.ts

```diff
@@ -644,3 +644,38 @@ export function getHighlightedText(text: string, highlightedText: string, matchD
 
     return spans;
 }
+
+/**
+ * Find an item from all Items based on path to the item, and expand all Items along the path.
+ * 
+ * @param {ItemData[]} pathToItem
+ * An arry of ItemData which represents the path to an item.
+ * 
+ * For example, in the following tree, pathToItem for D would be [A, B, D]
+ * - A
+ *   |- B
+ *      |- C
+ *      |- D
+ * 
+ * @param {ItemData} allItems
+ * An array of ItemData, which contains all the items. In the example above,
+ * allItems would be [A]
+ * 
+ * @return {boolean} true if an item is found, false otherwise
+ */
+export function findAndExpandItemByPath(pathToItem: ItemData[], allItems: ItemData[]): boolean {
+    let item: ItemData;
+
+    item = allItems.find(item =>
+        item.text == pathToItem[0].text && item.iconUrl == pathToItem[0].iconUrl
+    );
```

Comment:
Since text here is not fullyQualifiedName, but just the title of the item, it won't be sufficient as there are already items with the same title in one group.